src/config.rs

```diff
@@ -37,6 +37,35 @@ impl Default for SortOrder {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Hash, Copy, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum SassOutputStyle {
+    Nested,
+    Expanded,
+    Compact,
+    Compressed,
+}
+
+#[derive(Debug, PartialEq)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct SassOptions {
+    pub sass_dir: String,
```

Comment:
What were your thoughts on my suggesting we call this `import_dir`.  I feel like it'd make the semantics more clear.  This isn't a directly to hold your sass files.  This is a directory to hold the imported sass files.

Reply:
I only saw your comment after I pushed it all. Used the same name for compat-reasons, but don't have any feelings on any either name. Will change it.